save-frontend/README.md

```diff
@@ -7,4 +7,149 @@ To run frontend locally, use `./gradlew :save-frontend:browserDevelopmentRun --c
 
 To pack distribution, use `./gradlew :save-frontend:browserDevelopmentWebpack` and `./gradlew :save-frontend:browserProductionWebpack`.
 
-save-backend consumes frontend distribution as a dependency. Frontend distribution is copied and included in spring boot resources.
\ No newline at end of file
+save-backend consumes frontend distribution as a dependency. Frontend distribution is copied and included in spring boot resources.
+
+### `nginx` configuration
+The most interesting part of `nginx.conf` is here:
+```nginx configuration
+location / {
+  index index.html;
+  try_files $uri $uri/ /index.html;
+  add_header Cache-Control "private; no-store";
+  etag off;
+  add_header Last-Modified "";
+  if_modified_since off;
+}
+```
+
+Here we define a configuration for the `location /` block, which is the default location block in `nginx` and matches any
+request that doesn't match other specific location blocks.
+
+Let's break down each directive and explain what happens:
+
+1. `index index.html`;
+   This directive sets the default file to serve when a **directory** is requested.
+   In this case, if a request is made to a directory (e.g., http://example.com/), Nginx will try to serve the `index.html`
+   file from that directory. If the file doesn't exist, Nginx will move to the next directive.
+2. `try_files $uri $uri/ /index.html;`
+   This directive specifies a series of files and locations that Nginx should try to serve.
+   It acts as a fallback mechanism when the requested file is not found in the specified locations.
+- `$uri` represents the requested URI.
+- `$uri/` represents the URI followed by a trailing slash, typically used for directories.
+- `/index.html` is the last fallback option.
+3. `add_header Cache-Control "private; no-store";`
```

Comment:
is it done by purposes that we disable all caching?

Reply:
I guess yes, I guess, Peter tried to fix the caching issue with that